nixosConfigurations/hetzner-ephemeral-alpha/default.nix

```diff
@@ -270,9 +134,12 @@ in {
       Type = "simple";
     };
 
-    script = ''      ${pkgs.keep-core}/bin/keep-core \
+    script = ''      /var/mnt/keep-core/keep-client \
```

Comment:
Along with this.

Reply:
Keep-client is technically speaking a Go package, but it has some skeletons in the closet -- it uses NPM packages to add extra runtime information to the project source, which Go then uses to generate source code for the binary. It seems like the source code generation is dependent on specific versions of protobuf compilers. I'm afraid to use Nix to build the source, because it might deviate from the "source of truth". For this reason I thought it is a better idea to use an ad-hoc environment to install keep-client.

Let me add the environment under a new devShells directory.